sdk/communication/Azure.Communication.Chat/src/Models/ChatMessageReadReceipt.cs

```diff
@@ -1,26 +1,37 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+// <auto-generated/>
```

Comment:
This file is not autoregenerated. Seems like a copy/paste issue

Reply:
removed the line